src/operator/concat.cc

```diff
@@ -46,29 +46,39 @@ Operator* ConcatProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shap
 DMLC_REGISTER_PARAMETER(ConcatParam);
 
 MXNET_REGISTER_OP_PROPERTY(Concat, ConcatProp)
-.describe(R"code(Concate a list of array along a given axis.
+.describe(R"code(Join input arrays along the given dimension.
 
-The dimension sizes of the input arrays on the given axis should be the same.
+.. note:: `Concat` is deprecated use `concat`.
```

Comment:
You have two sentences (one declarative, one imperative). These need to be separated.

Sentence 1:
"`Concat` is deprecated 

Sentence 2:
use `concat`

Fix to:
`Concat` is deprecated. Use `concat` instead.